airbyte-integrations/connectors/source-pipedrive/source_pipedrive/spec.json

```diff
@@ -86,5 +81,13 @@
     }
   },
   "supportsIncremental": true,
-  "supported_destination_sync_modes": ["append"]
+  "supported_destination_sync_modes": ["append"],
```

Comment:
since this is a source, is that normal to set destination modes?

Reply:
oops, missed that, will remove